content/docs/guides/verify-jwt.md

```diff
@@ -62,32 +62,6 @@ The `issuer` and `audience` parameters should both be set to the domain of the *
   </div>
 </details>
 
-### PomeriumVerifier reference
```

Comment:
I think we should either remove the entire single-page web app section, or else switch it to use the server-side example from the js-sdk repo: https://github.com/pomerium/js-sdk/blob/main/examples/express/index.js.

Reply:
The easiest lift right now would probably be to remove that section altogether. Why swap the front-end example for a server-side example?

Reply:
Sure, sounds good to me.

Eventually I think it could make good sense to have both server-side Go and JS examples as those are the two SDKs we have right now.